src/resources/LivePhotos.ts

```diff
@@ -6,7 +6,7 @@ import { Method, Resource } from "../Resource";
 export type LivePhotoRequest = {
   applicantId: string;
   file: FileLike;
-  advancedValidation?: boolean;
+  advancedValidation?: string;
```

Comment:
why is can it not be boolean ? doesn't it make more sense as it is a flag?

Reply:
For this specific flag the backend expects it to be a string, I believe it's because this endpoint sends the params as `form-data` because of the file upload. If the content-type was sent as json the boolean flag would make more sense.

Reply:
I suppose we have to live with it then